### ISSUE TITLE
pip install datamule fails

### ISSUE THREAD

**Issue by Lanikeha** [john-friedman/datamule-python#29]
Hello,
I have installed Cython-3.0.11 but cannot install datamule.

Thanks for help!

```
PS C:\Users\krzys> pip install datamule
Collecting datamule
  Using cached datamule-0.418.tar.gz (1.7 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\krzys\AppData\Local\Temp\pip-install-tahb66pt\datamule_d44b7ff35f2949ad8c5fdc0b8f696ad4\setup.py", line 78, in <module>
          ext_modules=cythonize(
                      ^^^^^^^^^^
        File "C:\Users\krzys\AppData\Local\Programs\Python\Python312\Lib\site-packages\Cython\Build\Dependencies.py", line 1010, in cythonize
          module_list, module_metadata = create_extension_list(
                                         ^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\krzys\AppData\Local\Programs\Python\Python312\Lib\site-packages\Cython\Build\Dependencies.py", line 845, in create_extension_list
          for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
        File "C:\Users\krzys\AppData\Local\Programs\Python\Python312\Lib\site-packages\Cython\Build\Dependencies.py", line 117, in nonempty
          raise ValueError(error_msg)
      ValueError: 'datamule/parser/sgml_parsing/sgml_parser_cy.pyx' doesn't match any files
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed
```

**Comment by john-friedman** [comment]
Thanks for the post! Seems to be a version issue with python 3.12 (I use 3.11). Looking into it

**Comment by Lanikeha** [comment]
Thank you! I have installed 3.11 and it is installing correctly now.

I am a beginner python programmer so I will post one more question in a separate thread due to erros I get (I don't want to spam you and I have no idea if these errors are caused by my lack of knowledge)

**Comment by john-friedman** [comment]
Feel free to spam me. I want to make it easy to use SEC data, and you are one of my intended target demographics.

**Comment by john-friedman** [comment]
Just pushed an update. Should be fixed for python 3.12 and 3.13 now. Will test tomorrow.
